lib/main_containers.pm

```diff
@@ -148,7 +148,7 @@ sub load_host_tests_podman {
     # exclude rootless podman on public cloud because of cgroups2 special settings
     unless (is_openstack || is_public_cloud) {
         loadtest 'containers/rootless_podman';
-        loadtest 'containers/podmansh' unless (is_staging || is_leap("<16") || is_sle("<16") || is_sle_micro("<6.1") || is_leap_micro("<6.1"));
+        loadtest 'containers/podmansh' unless (is_staging || is_leap("<16") || is_sle("<16") || is_sle_micro("<6.2") || is_leap_micro("<6.2"));
```

Comment:
I think this should be constrained further as it's not available on SLEM 6.2 at  all: https://openqa.suse.de/tests/overview?distri=sle-micro&version=6.2&build=7.14&groupid=513

```
$ susepkg -p any podmansh | grep -e SLES/ -e Micro -e openSUSE
SLES/15.5 podmansh 4.9.5-150500.3.40.1
SLES/15.6 podmansh 4.9.5-150500.3.40.1
SLES/15.7 podmansh 4.9.5-150500.3.40.1
SLES/16.0 podmansh 5.2.5-160000.4.1
SLE-Micro/5.5 podmansh 4.9.5-150500.3.40.1
openSUSE_Leap/15.6 podmansh 4.9.5-150500.3.40.1
```
openSUSE_Tumbleweed podmansh 5.4.2-1.1

Reply:
SLEM 6.2 is still a product in development, while SLEM 6.0 and 6.1 are released and maintained.

I would prevent those test issues in the maintenance pipeline, while they are acceptable in new product validation.